src/app/(markets)/markets/[address]/components/clob/orderbook-table-large.tsx

```
@@ -0,0 +1,323 @@
+                        6
+                      )}
+                    </Text>
+                  </HStack>
+                  <HStack w='144px' h='full' justifyContent='flex-end'>
+                    <Text {...paragraphRegular}>
+                      {NumberUtil.toFixed(item.cumulativePrice, 6)} {market?.collateralToken.symbol}
+                    </Text>
+                  </HStack>
+                </HStack>
+              ))}
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)